content/zh/docs/chart_template_guide/builtin_objects.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "内置对象"
+description: "模板可用的内置对象"
+weight: 3
+---
+
+&emsp;&emsp;对象从模板引擎传递到模板中。 你的代码也可以传递对象。（我们在使用`with`和`range`语句时，会看到示例）。甚至于有几种方式可以在你的模板中创建新对象，比如说我们后面会看到的`tuple`功能。
+
+&emsp;&emsp;对象可以是简单的且仅有一个值。或者可以包含其他对象或方法。比如，`Release`对象包含一些对象（比如：`Release.Name`）和`Files`对象有一组方法。
+
+&emsp;&emsp;在上一部分中，我们用`{{ .Release.Name }}`在模板中插入版本名称。`Release`是你可以在模板中访问的高级对象之一。 
+
+- `Release`： 该对象描述了版本发布本身。包含了以下对象：
+  - `Release.Name`： release名称
+  - `Release.Namespace`： 版本中包含的命名空间(如果manifest没有覆盖的话)
+  - `Release.IsUpgrade`： 如果当前操作是升级或回滚的话，需要将该值设置为`true`
+  - `Release.IsInstall`： 如果当前操作是安装的话，需要将该值设置为`true`
+  - `Release.Revision`： 此次修订的版本号。安装时是1，每次升级或回滚都会自增
+  - `Release.Service`： 该service用来渲染当前模板。Helm里一般是`Helm`
+- `Values`： Values是从`values.yaml`文件和用户提供的文件传进模板的。`Values`默认为空
+- `Chart`： `Chart.yaml`文件内容。 `Chart.yaml`里的任意数据在这里都可以可访问的。比如 `{{ .Chart.Name }}-{{ .Chart.Version }}` 会打印出 `mychart-0.1.0`
+  - [Chart 指南](http://helm.sh/zh/docs/topics/charts.md#Chart-yaml-文件) 中列出了可用字段
+- `Files`： 在chart中提供访问所有的非特殊文件。当你不能使用它访问模板时，你可以访问其他文件。请查看这个部分 [文件访问](http://helm.sh/docs/chart_template_guide/accessing_files.md) 了解跟多信息
```

Comment:
I recommend removing the `http://helm/sh` part of these URLs (especially in this one as it will hard-code the link to always use English).

Reply:
I won't block merging this PR, but FYI we will want to get that fixed in most of these files.

Reply:
OK，I'll fix it all.